packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraSettingsTests.m

```diff
@@ -0,0 +1,244 @@
+  _minFrameDurationExpectation = [test expectationWithDescription:@"minFrameDurationExpectation"];
+  _maxFrameDurationExpectation = [test expectationWithDescription:@"maxFrameDurationExpectation"];
+  _beginConfigurationExpectation =
+      [test expectationWithDescription:@"beginConfigurationExpectation"];
+  _commitConfigurationExpectation =
+      [test expectationWithDescription:@"commitConfigurationExpectation"];
+  _audioSettingsExpectation = [test expectationWithDescription:@"audioSettingsExpectation"];
+  _videoSettingsExpectation = [test expectationWithDescription:@"videoSettingsExpectation"];
+
+  return self;
+}
+
+- (BOOL)lockDevice:(AVCaptureDevice *)captureDevice error:(NSError **)outError {
+  [_lockExpectation fulfill];
+  return YES;
+}
+
+- (void)unlockDevice:(AVCaptureDevice *)captureDevice {
+  [_unlockExpectation fulfill];
+}
+
+- (void)beginConfigurationForSession:(AVCaptureSession *)videoCaptureSession {
+  [_beginConfigurationExpectation fulfill];
+}
+
+- (void)commitConfigurationForSession:(AVCaptureSession *)videoCaptureSession {
+  [_commitConfigurationExpectation fulfill];
+}
+
+- (void)setMinFrameDuration:(CMTime)duration onDevice:(AVCaptureDevice *)captureDevice {
+  if (duration.value == 10 && duration.timescale == gTestFPS * 10) {
+    [_minFrameDurationExpectation fulfill];
+  }
+}
+
+- (void)setMaxFrameDuration:(CMTime)duration onDevice:(AVCaptureDevice *)captureDevice {
+  if (duration.value == 10 && duration.timescale == gTestFPS * 10) {
+    [_maxFrameDurationExpectation fulfill];
+  }
+}
+
+- (AVAssetWriterInput *)assetWriterAudioInputWithOutputSettings:
+    (nullable NSDictionary<NSString *, id> *)outputSettings {
+  if ([outputSettings[AVEncoderBitRateKey] isEqual:@(gTestAudioBitrate)]) {
+    [_audioSettingsExpectation fulfill];
+  }
+
+  return [AVAssetWriterInput assetWriterInputWithMediaType:AVMediaTypeAudio
+                                            outputSettings:outputSettings];
+}
+
+- (AVAssetWriterInput *)assetWriterVideoInputWithOutputSettings:
+    (nullable NSDictionary<NSString *, id> *)outputSettings {
+  if ([outputSettings[AVVideoCompressionPropertiesKey] isKindOfClass:NSMutableDictionary.class]) {
+    NSDictionary *compressionProperties = outputSettings[AVVideoCompressionPropertiesKey];
+
+    if ([compressionProperties[AVVideoAverageBitRateKey] isEqual:@(gTestVideoBitrate)] &&
+        [compressionProperties[AVVideoExpectedSourceFrameRateKey] isEqual:@(gTestFPS)]) {
+      [_videoSettingsExpectation fulfill];
+    }
+  }
+
+  return [AVAssetWriterInput assetWriterInputWithMediaType:AVMediaTypeVideo
+                                            outputSettings:outputSettings];
+}
+
+- (void)addInput:(AVAssetWriterInput *)writerInput toAssetWriter:(AVAssetWriter *)writer {
+}
+
+- (NSMutableDictionary *)recommendedVideoSettingsForOutput:(AVCaptureVideoDataOutput *)output {
+  return [NSMutableDictionary new];
```

Comment:
https://google.github.io/styleguide/objcguide.html#do-not-use-new

Reply:
replaced with alloc-init